inc/dashboard/namespace.php

```diff
@@ -0,0 +1,421 @@
+<?php
+/**
+ * Dashboard Analytics
+ *
+ * @package aws-analytics
+ */
+
+namespace Altis\Analytics\Dashboard;
+
+use Altis\Analytics\Blocks;
+use Altis\Analytics\Utils;
```

Comment:
Nicer to use WP_Query and add the type hint to the callback.
```suggestion
use Altis\Analytics\Utils;
use WP_Query;
```